app/api/api_v1/endpoints/auth.py

```
@@ -127,21 +79,32 @@ async def google_auth(
 
 
 @router.post("/complete-profile", response_model=CompleteProfileResponse)
-def complete_profile(
-    request: CompleteProfileRequest,
-    db: Session = Depends(get_db),
+async def complete_profile(
+    current_user: VerifiedUserDep,
+    session: SessionDep,
+    request: CompleteProfileRequest = Form(media_type="multipart/form-data"),
```

Ruff (0.12.2)

Do not perform function call `Form` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)